core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -3010,12 +3010,12 @@ public RecoveryRequestResponse call(TrxRegionService instance) throws IOExceptio
               rresult = table.coprocessorService(TrxRegionService.class, startKey, endKey, callable);
             }
             catch (ServiceException se) {
-                LOG.error("Exception thrown when calling coprocessor: ", se);
-                throw new IOException("Problem with calling coprocessor, no regions returned result", se);
+                LOG.error("Service exception thrown when recoveryRequest: ", se);
```

Comment:
Reading the text, I see, "Service exception thrown when recoveryRequest". I'm left wondering, "when recoveryRequest what?" Is that information packaged inside the se object?